### PR TITLE
Added credits section to our app

### PR DESCRIPTION
**Pull request by ChenMorSays**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -34,6 +34,38 @@ <h1 class="display-2 main-title" [class.mobile-title]="isMobile"><strong>My Holo
   </div>
 </section>
 
+<strong>
```
Comment (NirMekin) [inline src/app/app.component.html:37]: strong should not contain block element

File: src/app/app.component.html [hunk 2]
```diff
@@ -34,6 +34,38 @@ <h1 class="display-2 main-title" [class.mobile-title]="isMobile"><strong>My Holo
...
+  <div class="row">
+    <div class="col-3">
+      <a class="credits" target="_new" href="//twitter.com/ChenMorSays">
```
Comment (NirMekin) [inline src/app/app.component.html:40]: also a tag change it to be background-image

File: src/app/app.component.html [hunk 3]
```diff
@@ -34,6 +34,38 @@ <h1 class="display-2 main-title" [class.mobile-title]="isMobile"><strong>My Holo
...
+    <div class="col-3">
+      <a class="credits" target="_new" href="//www.linkedin.com/in/nir-mekin-6b20bb117/">
+        <div class="credits-pos-3"> Nir
+          <img src="//media-exp1.licdn.com/dms/image/C5103AQFOqBnoPX59dQ/profile-displayphoto-shrink_800_800/0?e=1586995200&v=beta&t=u3PlPOB5Osu5FUSh8mLx_323y9RDEaj6ZP9-pPaVqKM">
+        </div>
+      </a>
+    </div>
+    <div class="col-3">
+      <a class="credits" target="_new" href="//www.linkedin.com/in/naor-haimov-045008a9/">
+        <div class="credits-pos-4"> By Naor
```
Comment (NirMekin) [inline src/app/app.component.html:62]: 'By' should not be a part of the link of the name

File: src/app/app.component.css [hunk 4]
```diff
@@ -1,3 +1,50 @@
...
+.credits-pos-3 {
+  right: 165px
+}
+
+.credits-pos-4 {
+  right: 231px
+}
+
+.credits div {
+  /*right: 0;*/
```
Comment (NirMekin) [inline src/app/app.component.css:24]: remove

File: src/app/app.component.css [hunk 5]
```diff
@@ -1,3 +1,50 @@
...
+  bottom: 0;
+  position: fixed;
+  padding: 0.5em;
+  font-weight: 600;
+  z-index: 40;
```
Comment (NirMekin) [inline src/app/app.component.css:29]: why you need z-index? check if it can be removed

File: src/app/app.component.css [hunk 6]
```diff
@@ -1,3 +1,50 @@
...
+  vertical-align: middle;
+  background-color: #fc4946;
```
Comment (NirMekin) [inline src/app/app.component.css:31]: this will be ignored, remove

File: src/app/app.component.css [hunk 7]
```diff
@@ -1,3 +1,50 @@
...
+  color: #42b4ff;
```
Comment (NirMekin) [inline src/app/app.component.css:32]: also this

**Review by NaorHai — APPROVED** [review]

File: src/app/app.component.html [hunk 8]
```diff
@@ -34,6 +34,38 @@ <h1 class="display-2 main-title" [class.mobile-title]="isMobile"><strong>My Holo
   </div>
 </section>
 
+<strong>
+  <div class="row">
+    <div class="col-3">
+      <a class="credits" target="_new" href="//twitter.com/ChenMorSays">
+        <div class="credits-pos-1"> Chen
```
Comment (NirMekin) [inline src/app/app.component.html:41]: we shouldn't give class to every element if it should be unique use 'id' instead but this is not the way

**Review by NirMekin — APPROVED** [review]

**Pull request merged**